app/models/version.rb

```diff
@@ -20,13 +20,26 @@ def numbers_must_increase
     errors.add(:number, message)
   end
 
+  def draft_and_live_versions
+    result = Queries::DraftAndLiveVersions.call(target, self.class.table_name)
+    if result["draft"] == self.number
+      draft_version = self.number
+      live_version = result["live"]
+    elsif result["live"] == self.number
+      draft_version = result["draft"]
+      live_version = self.number
+    end
```

Comment:
So I got a little caught on what happens here when neither the `if` or `elsif` are called and as to how this returns `[nil, nil]` when neither of the variables seem to be defined - which is a quirk of ruby scoping rules: http://stackoverflow.com/questions/12614805/why-can-i-refer-to-a-variable-outside-of-an-if-unless-case-statement-that-never

I think if we defined them before the conditional or had an else it'd be clearer for a future reader. I know this is just code that was moved, so could resolve this outside of this PR.